### PR TITLE
comparing sim data with raw data

### PR DESCRIPTION
**Pull request by mohamedmonem123**
comparing sim data with raw data

### CODE REVIEW

File: notebooks/pad_documentation/.~lock.Pad_documentation _for _SKY130_ MPW_Manufacturing.csv#
Comment (atorkmabrains) [inline notebooks/pad_documentation/.~lock.Pad_documentation _for _SKY130_ MPW_Manufacturing.csv#:1]: @mohamedmonem123 Please remove that file.

File: notebooks/templates/nfet_01v8_template.cir
Comment (atorkmabrains) [inline notebooks/templates/nfet_01v8_template.cir:1]: @mohamedmonem123 Please remove the control statement and use raw data instead.

**Review by atorkmabrains — APPROVED** [review]

**Comment by mohamedmonem123** [comment]
@atorkmabrains 
Ready for review

**Review by atorkmabrains — APPROVED** [review]
Thanks @mohamedmonem123

**Pull request merged**